### PR TITLE
git-subcopy: init at 0.1.0

### PR DESCRIPTION
**Pull request by jD91mZM2**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- x ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by jD91mZM2** [comment]
Not sure, but I feel like a bump is in order?

**Comment by doronbehar** [comment]
> Not sure, but I feel like a bump is in order?

That doesn't help as no one is "watching" this repo because of the enormous amount of PRs.

**Comment by doronbehar** [comment]
Also, you should use "init at" and not "new at" in your PR title, as explained in [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

File: pkgs/development/tools/git-subcopy/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{ callPackage, lib, fetchFromGitLab }:
+
+let
+  pname = "git-subcopy";
+  version = "0.1.0";
+
+  package = (callPackage ./Cargo.nix {}).rootCrate.build;
+in package.overrideAttrs (attrs: {
```
Comment (doronbehar) [inline pkgs/development/tools/git-subcopy/default.nix:8]: How did you generate that `Cargo.nix`? And why not use `rustPlatform`?

**Comment by jD91mZM2** [comment]
I no longer care about this project, let's not put it in nixpkgs :)

**Pull request closed without merging**